security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -178,7 +183,7 @@ func (s *sdsservice) Generate(_ *model.Proxy, _ *model.PushContext, w *model.Wat
 			names = append(names, i.Name)
 		}
 	}
-	resp, err := s.generate(names)
+	resp, err := s.generate(names, s.pkpConf)
```

Comment:
Do you need to pass in `s.pkgConf` as you are calling a receiver on `s` anyway?

Reply:
Done